scripts/check-config.js

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const fs = require('fs');
+const path = require('path');
+
+const RELATIVE_FILE = 'server/config/local.json';
+const CONFIG_FILE = path.resolve(__dirname, '..', RELATIVE_FILE);
+
+if (!fs.existsSync(CONFIG_FILE)) {
```

Comment:
Here also check if `process.env.CONFIG_FILES` exists. 
  if it exists (defined) then do **not** copy the local.json file. This is an extra precaution, CONFIG_FILES var overrides everything else

See similar example here:
https://github.com/mozilla/fxa-content-server/blob/f75b7165f19055809a4ed28ce0091ebc53575f2b/grunttasks/selectconfig.js#L21